buildozer/__init__.py

```diff
@@ -335,6 +346,7 @@ def cmd(self, command, **kwargs):
         process.communicate()
         if process.returncode != 0 and break_on_error:
             self.error('Command failed: {0}'.format(command))
+            self.log_env(1, kwargs['env'])
```

Comment:
Thanks @mahomahomaho I think that PR makes sense. I have a couple of minor questions however.
Why is the log level hard-coded to two different values, once with `2` and then here with `1`?
Also why aren't we consistent in the way to access `env` key in kwargs? Once with `kwargs.get("env")` and then with `kwargs['env']`?

Reply:
yes, you're right. It should be kwargs['env'] in both cases, because we're sure that kwargs contains 'env' key.

Reply:
regarding two different levels - It's because in this place, you will get info in the same level as msg aabove (error). 

I was thinking about various approachs, and I considered to define log_debug_env and log_info_env ... but I decided to not make too many methods. But indeed - 1 and 2 could be somehow ENUMed.